model/post.go

```diff
@@ -31,6 +31,7 @@ type Post struct {
 	Id            string          `json:"id"`
 	CreateAt      int64           `json:"create_at"`
 	UpdateAt      int64           `json:"update_at"`
+	EditAt        int64           `json:"edit_at"`
```

Comment:
@jwilander, Yes, it looks like `OriginalId` of a post is only udpated from `Update` method in post sql store. And it's initial value is set to empty string in `Save` (inside `PreSave`) method. I'll use that field and remove the EditAt field.